scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-9.yaml

```diff
@@ -0,0 +1,212 @@
+- job:
+    name: cloud-mkphyscloud-qa-scenario-9
+    node: cloud-mkphyscloud-gate-qa
+    description: |
+      <b>This job is managed by JJB! Changes must be done in
+      <a href='https://github.com/SUSE-Cloud/automation/tree/master/scripts/jenkins/qa/'>git</a>
+      </b>
+      This job will redeploy scenario-9:
+        - 8 nodes
+        - HA with SBD (2 nodes)
+        - ceph (3 nodes)
+        - 3 kvm compute nodes
```

Comment:
All the other jobs seem to use 2 compute nodes. Is there a special reason for needing 3 here? Also as far as I can see here this is using the ```hw_number``` 2 are you sure that that setup really has enough nodes? @vmoravec could you clarify on this?